bitvec.cabal

```diff
@@ -55,11 +55,10 @@ source-repository head
   type: git
   location: git://github.com/Bodigrim/bitvec.git
 
-flag libgmp
+flag simd
   description:
-    Link against the GMP library for the ultimate performance of
-    `zipBits`, `invertBits` and `countBits`. Users are strongly encouraged
-    to enable this flag whenever possible.
+    Use a C SIMD implementation for the ultimate performance of `zipBits` and `invertBits`.
+    Users are strongly encouraged to enable this flag whenever possible.
   default: False
```

Comment:
Shall we enable it by default?

Reply:
Hmm, wouldn't that break with the JS or WASM backend (or in the rare case that the default C compiler doesn't support OpenMP)?

Reply:
I would not worry about them, they are too boutique. Until there is a CI job to witness the contrary, we can effectively assume that they are broken anyways.

Flipping a package flag is likely the least of obstacles for GHC-to-JS compilation right now. We can potentially say `if flag(openmp) && !arch(javascript) ...`, if Cabal 3.10 already supports it.